cmd/influxd/run.go

```diff
@@ -682,6 +669,15 @@ func joinServer(s *influxdb.Server, u url.URL, joinURLs []url.URL) {
 			return
 		}
 	}
+
+	if len(joinURLs) == 0 {
```

Comment:
No.  Should not return here because this is looping over each joinURL.  If one fails, it will try the next one.  If they all fail, L681 will run and fatal out.